pages/bookmarks.html

```diff
@@ -5,7 +5,31 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="../styles/index.css">
-    <title>Document</title>
+    <title>Bookmarks</title>
   </head>
-  <body></body>
+    <body class="app-grid">
+    <h1 class="header">Bookmarks</h1>
```

Comment:
all other pages have a h1 element with the additional class headings. did you forget it here?